src/features/defi/providers/yearn/components/YearnManager/Withdraw/components/Status.tsx

```diff
@@ -56,14 +57,13 @@ export const Status = () => {
   const feeAsset = useAppSelector(state => selectAssetById(state, feeAssetId))
   const feeMarketData = useAppSelector(state => selectMarketDataById(state, feeAssetId))
 
-  const accountSpecifier = useAppSelector(state =>
-    selectFirstAccountSpecifierByChainId(state, chainId),
-  )
+  const accountAddress = useMemo(() => fromAccountId(accountId ?? '').account, [accountId])
 
   const serializedTxIndex = useMemo(() => {
-    if (!(state?.txid && state?.userAddress)) return ''
-    return serializeTxIndex(accountSpecifier, state.txid, state.userAddress)
-  }, [state?.txid, state?.userAddress, accountSpecifier])
+    if (!(state?.txid && accountId && accountAddress)) return ''
+    if (!(state?.txid && accountId && accountAddress?.length)) return ''
```

Comment:
do we need two lines here?

Reply:
512176076cd50f7548cfaa4a62bf1d92ba63e6f5